stl/inc/valarray

```diff
@@ -1264,9 +1304,15 @@ public:
 
     indirect_array() = delete;
 
-    indirect_array(const indirect_array&); // not defined
+    indirect_array(const indirect_array&) = default;
 
-    indirect_array& operator=(const indirect_array&); // not defined
+    indirect_array& operator=(const indirect_array& _Right) {

```

Comment:
Ditto ([\[indirect.array.assign\]](http://eel.is/c++draft/indirect.array.assign))
```suggestion
    const indirect_array& operator=(const indirect_array& _Right) const {
```